src/main/java/weather/CurrentWeatherJSONComment.java

```diff
@@ -0,0 +1,49 @@
+package weather;
+
+public class CurrentWeatherJSONComment
```

Comment:
You don't need to put this as a .java class you can put this in a .txt or better yet as a .json